ui/src/app/certificate_requests/row.tsx

```diff
@@ -100,9 +109,17 @@ export default function Row({ id, csr, certificate, ActionMenuExpanded, setActio
                     </span>
                 </td>
                 <td id="expanded-row" className="p-table__expanding-panel" aria-hidden={detailsMenuOpen ? "false" : "true"}>
-                    <div className="row">
-                        <div className="col-8">
-                            <p><b>Common Name</b>: {csrObj.subjects.find((e) => e.type == "Common Name")?.value}</p>
+                    <div className="col-8">
+                        <div className="certificate-info">
+                            <h4>CSR Details</h4>
+                            {getFieldDisplay("Common Name", csrObj.commonName)}
+                            {getFieldDisplay("Subject Alternative Name DNS", csrObj.sansDns && csrObj.sansDns.length > 0 ? csrObj.sansDns.join(', ') : "")}
+                            {getFieldDisplay("Subject Alternative Name IP addresses", csrObj.sansIp && csrObj.sansIp.length > 0 ? csrObj.sansIp.join(', ') : "")}
+                            {getFieldDisplay("Organization", csrObj.organization)}
+                            {getFieldDisplay("Country", csrObj.country)}
+                            {getFieldDisplay("Locality", csrObj.locality)}
```

Comment:
We have "Country" and "Locality" but not "State or Province", I think we should add this one as well

Reply:
State or Province added.